src/components/ProductItem/ProductItem.jsx

```diff
@@ -0,0 +1,24 @@
+import './product-item.css';
+
+const ProductItem = ({ product }) => {
+    return (
+        <div class="product-card" key={product.id}>
+            <div>
+                <div class="card-top-half">
+                    <i class="fas fa-regular fa-heart product-fav-icn fav-active"></i>
+                    <div className="product-asset-container">
+                        <img src={product.asset} alt="product body asset" class="product-body-asset" />
+                    </div>
+                    
+                    <div class="card-top-half-text">
+                        <p class="product-item-title">{product.title}</p>
+                        <p class="product-item-price">Rs {product.price}</p>
+                    </div>
+                </div>
+                <button class="product-item-cta">View Product</button>
+            </div>
+        </div>
+    );
+};
+
```

Comment:
you can destructrue `products` prop inside the component like
`const {asset,title,price}=products` and use the destructured values whenever required instead of using `product.somvalue`